src/UserCreator/reducer.js

```diff
@@ -1,13 +1,17 @@
-import { UPDATE_NAME_FIELD } from "./actions";
+import { UPDATE_NAME_FIELD, CREATE_USER } from "./actions";
 
 const initialState = {
-  name: ""
+  name: "",
+  // Only way of getting the information back to the component..
+  createdUser: null
```

Comment:
Only way of "listening" actions on the React component level. Probably the biggest drawback with this approach so far